application/src/main/java/org/opentripplanner/model/TripTimeOnDate.java

```diff
@@ -180,14 +193,17 @@ public TripTimes getTripTimes() {
     return tripTimes;
   }
 
-  public int getStopCount() {
-    return tripTimes.getNumStops();
-  }
-
   public int getScheduledArrival() {
     return tripTimes.getScheduledArrivalTime(stopIndex);
   }
 
+  /**
+   * Returns the scheduled arrival as an Instant.
+   */
+  public Instant scheduledArrival() {
+    return toInstant(getScheduledArrival());
+  }
+
```

Comment:
I am unsure if this should be here - it is API specific. Also, I do not think the implementation is right. It passes the getScheduledArrival() in - witch is not EPOC. 

This is mapping code that belong in the UI - at least until we must support more han one time-zone.